modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/SourceModelParser.java

```diff
@@ -152,6 +150,8 @@ public interface SourceModelParser extends SemanticTermsParser, StereotypeModelP
 
   Optional<SinceMuleVersionModelProperty> getSinceMuleVersionModelProperty();
 
+  Optional<String> getMinMuleVersion();
```

Comment:
With these changes, the parser will now be able to parse the min mule version of the components. Is there any issue created to add this notion to the model and make the model loader populate it?

Reply:
This issue https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000lfXIeYAM/view handles comparing the parsed min mule version against the pom (this would be the model loader responsibility).
While https://gus.lightning.force.com/lightning/r/a07EE00000lfQ0mYAE/view will explore how to add this information to the model (if via model property, or as a new element, etc.)

Reply:
Got it 👌  , I think that the last part should be covered by https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000011Dq4hYAC/view this issue, which was closed based on the assumption that we would use the SinceMuleVersionModelProperty. Don't know if we should re open that one, or create a follow-up issue.

Reply:
I think we should create a new one, after designing the solution in the spec.

Reply:
I feel that the "where to put that info in the model" is out of the scope of the pruning spec.